docs/local-development.md

```diff
@@ -12,8 +12,8 @@ The `sam local invoke` command invokes your function in a Docker container. You
 For example, given this function:
 
 ```php
-λ(function ($event) {
-    return 'Hello ' . ($event['name'] ?? 'world!');
+lambda(function (array $event) {
```

Comment:
Good point!

However you shouldn't type-hint with `array` since the variable can be anything, including `null` which is what we use in the example below.

Reply:
My bad the event is always an array :) (I just realized that all the examples are like this.